lib/hotel.rb

```diff
@@ -0,0 +1,119 @@
+# hotel
+require_relative 'room.rb'
+require 'date'
+
+module HotelManagment
+  class Hotel
+    attr_reader :rooms, :reservations, :found_reservations, :unreserved_rooms, :blocks
+    def initialize
+      @rooms = []
```

Comment:
If you're setting `@rooms` to `[]`  in initialize, why do you have an `add_20_rooms` method.  I would suggest instead making `add_20_rooms` private and calling it from `initialize`.  I would also suggest renaming it to something less specific as to the implementation (number of rooms).  That way you could extend or subclass `Hotel` and override the method to change the number of rooms.